crates/core/tedge_mapper/src/c8y/converter.rs

```diff
@@ -23,14 +23,17 @@ use c8y_smartrest::{
     },
 };
 use c8y_translator::json;
+
+use logged_command::LoggedCommand;
 use mqtt_channel::{Message, Topic, TopicFilter};
+use plugin_sm::operation_logs::OperationLogs;
```

Comment:
It seems that this `use` is the only reason that makes you move the `plugin_sm` out of the agent. 

If so, I do think that the `plugin_sm` should that stay internal to the agent and that only the `operation_logs` module should be promoted to a crate. Furthermore, can you consider to have a single crate for `operation_logs` and `logged_command`.